src/LibraryViewExtension/Handlers/SearchResultDataProvider.cs

```diff
@@ -40,6 +40,8 @@ public override Stream GetResource(IRequest request, out string extension)
             var pathAndQuery = uri.PathAndQuery;
             var index = url.IndexOf(serviceIdentifier);
             var text = url.Substring(index + serviceIdentifier.Length + 1);
+            // unescape query before searching
+            text = Uri.UnescapeDataString(text);
```

Comment:
so, this accepts all special chars?

Reply:
I believe so, here we unescape the encoded URI sent from Librarie.js before searching